lectures/solow.md

```diff
@@ -175,15 +173,15 @@ def plot45(kstar=None):
 plot45()
 ```
 
-Suppose, at some $k_t$, the value $g(k_t)$ lies strictly above the 45 degree line.
+Suppose, at some $k_t$, the value $g(k_t)$ lies strictly above the 45-degree line.
 
 Then we have $k_{t+1} = g(k_t) > k_t$ and capital per worker rises.
 
 If $g(k_t) < k_t$ then capital per worker falls.
 
 If $g(k_t) = k_t$, then we are at a **steady state** and $k_t$ remains constant.
 
-(A steady state of the model is a [fixed point](https://en.wikipedia.org/wiki/Fixed_point_(mathematics)) of the mapping $g$.)
+(A [steady state](https://intro.quantecon.org/scalar_dynam.html#steady-states) of the model is a [fixed point](https://en.wikipedia.org/wiki/Fixed_point_(mathematics)) of the mapping $g$.)
```

Comment:
@SylviaZhaooo rather than a web link it would be good to use an internal link by:

1. adding a tag such as `(scalar-dynam:steady-state)=` as a reference marker in `scalar_dynam.md` at the intended location
2. reference it here using 
```
{ref}`steady state <scalar-dynam:steady-state>`
```